README.md

```diff
@@ -26,3 +26,5 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+Answer: The runtime fo the algorithm I implemented is $\Theta$ (nlog n). This is because the algorithm is very similar to similar to the merge sort algorithm. The only difference is that the array is broken up into 3 differnt arrays at each recursive step instead of 2 and the elements in the array are added instead of compared and swapped. The recurence relation would be T(n) = { 1 if n <= 1,  3T(n/3) + n if n > 1}. 
```

Comment:
Is there something wrong with the way I am naming the file or committing the pull request? For all the exercises I have submitted it says that my code fails the test but when I use the test in my own code it works. I am not sure if this is because my code is wrong or something is causing it to fail the test when I commit it to GitHub.

Reply:
I don't think so. There should be a problem with your code that you missed some pre-conditions on your code. We can work on it Wednesday in the lab.